src/overlays/actors/ovl_En_Karebaba/z_en_karebaba.c

```diff
@@ -1,6 +1,10 @@
 #include "z_en_karebaba.h"
 
-#define FLAGS 0x00000005
```

Comment:
Why get rid of the #define? It seems standard.

Reply:
Wasn't really sure what it was for, to be honest. I can add it back in.